front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -457,14 +457,16 @@ const Moderation = memo<Props & InjectedIntlProps>(({ className, intl }) => {
               placement="right"
             />
           </PageTitleWrapper>
-          <Button
-            buttonStyle="secondary"
-            onClick={openSettingsModal}
-            locale={locale}
-            icon="settings"
-          >
-            <FormattedMessage {...messages.settings} />
-          </Button>
+          {profanityBlockerSetting && (
```

Comment:
They will not be able to launch the modal if the setting is not present

Reply:
With the second setting added in i2, I'll make it a `profanityBlockerSetting `|| `inappropriateContentSetting`